projects/S905/patches/linux/linux-0005-add_dummy_wifi_driver.patch

```diff
@@ -0,0 +1,74 @@
+From 1fb79b55b66c429ee261fbb2cf377b0d3b9b0ee5 Mon Sep 17 00:00:00 2001
```

Comment:
Didn't we merge that dummy wifi driver in linux-amlogic ;)
https://github.com/LibreELEC/linux-amlogic/commit/d34db48cc26c967850e8bf7e89ef49c3be367dc4

Reply:
We did but until the package is bumped its needed here

Reply:
Ahh right sorry I forgot